app/models/user.rb

```diff
@@ -1,4 +1,11 @@
 class User < ApplicationRecord
+  before_validation :set_default
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  # :confirmable
```

Comment:
- [ ] As requested by the previous reviewer, and according to the requirements of this milestone, please implement the `confirmable` module in your devise config.

You already have the required columns for it to work in your migrations so the only thing that's left is including `:confirmable` on this list of symbols.

Please consider implementing this! 🤓 🔧